cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/transformers/SeqStepsTransformer.java

```diff
@@ -119,6 +123,17 @@ private void validateWaitStep(Map<String, String> stepProps) {
         if (StringUtils.isEmpty(args) || StringUtils.equals("\"\"", args)) {
             throw new RuntimeException(WAIT_PARAM_REQUIRED);
         }
+        // we need to check only constant value. It's wrapped into double quotes.
```

Comment:
remove comments, the code is self explained
extract this a method -> validateWaitArgRange